src/github.com/getlantern/flashlight/main/settings_test.go

```diff
@@ -11,6 +11,6 @@ func TestNotPersistVersion(t *testing.T) {
 	version := "version-not-on-disk"
 	revisionDate := "1970-1-1"
 	buildDate := "1970-1-1"
-	Load(version, revisionDate, buildDate)
+	LoadSettings(version, revisionDate, buildDate)
```

Comment:
This was a bonus.  I noticed that this test was failing because it was using the wrong name for this method.